src/psyclone/tests/dynamo0p3_transformations_test.py

```diff
@@ -4573,13 +4577,22 @@ def test_rc_no_loop_decrease():
             "transformation does nothing") in str(excinfo)
 
 
-def test_rc_remove_halo_exchange(tmpdir, f90, f90flags):
-    ''' Test that a halo exchange is removed if redundant computation means
-    that it is no longer required '''
+def test_rc_remove_halo_exchange(tmpdir, f90, f90flags, monkeypatch):
+    '''Test that a halo exchange is removed if redundant computation means
+    that it is no longer required. Halo exchanges are not required in
+    this example when we compute annexed dofs by default. Therefore we
+    revert to computing over dofs by default (via monkeypatch) to
+    perform the test.
+
+    '''
+    import psyclone.config
+    monkeypatch.setattr(psyclone.config, "COMPUTE_ANNEXED_DOFS", False)
     _, info = parse(os.path.join(
         BASE_PATH, "14.7_halo_annexed.f90"),
                     api=TEST_API)
     psy = PSyFactory(TEST_API).create(info)
+    invoke = psy.invokes.invoke_list[0]
```

Comment:
Not sure what happened here but these two new lines are duplicates of those below (just above where `rc_trans` is assigned to).

Reply:
Apologies, remnants from debugging I think.